modelci/hub/loader.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Author: Li Yuanming
+Email: yli056@e.ntu.edu.sg
+Date: 2/17/2021
+
+Load model from path to runnable object
```

Comment:
give a more detailed description 

Seems this file contains functions about model loading 

It should be clarified clearly clearly the differences among deployer, data loader & model loader 

So I recommend to rename the file

Reply:
Renamed to `model_loader`.